.ci/premerge_advisor_upload.py

```diff
@@ -12,9 +12,10 @@
 
 import generate_test_report_lib
 
-PREMERGE_ADVISOR_URL = (
-    "http://premerge-advisor.premerge-advisor.svc.cluster.local:5000/upload"
-)
+PREMERGE_ADVISOR_URLS = [
+    "http://34.82.126.63:5000/upload",
```

Comment:
If these addresses have names associated with them, putting those in a comment might be nice.

Reply:
Added. We should use domain names eventually to identify them and enable OpenSSL, but the internal process to get domain names is rather complicated.